spk/sickchill/Makefile

```diff
@@ -0,0 +1,40 @@
+SPK_NAME = sickchill
+SPK_VERS = $(shell date +%Y%m%d)
+SPK_REV = 0
+SPK_ICON = src/sickchill.png
+
+SPK_DEPENDS = "python3>=3.6:git"
+
+MAINTAINER = Nyaran
+DESCRIPTION = Automatic Video Library Manager for TV Shows. It watches for new episodes of your favorite shows, and when they are posted it does its magic.
+DESCRIPTION_SPN = Gestor automático para bibliotecas de series. Busca nuevos episodios de tus series favoritas, y cuando son publicados hace su magia.
+RELOAD_UI = yes
```

Comment:
```suggestion

```

IRC This is not necessary, This only used when the GUI is directly integrated with the DSM (like the native apps).